### PR TITLE
feat #SH-35: As a logged in user, I want to view all my groups on the Portal

### PR DESCRIPTION
**Pull request by Nik720**
- added new module for groups
- added new menu My Groups

### CODE REVIEW

File: src/app/client/src/app/modules/core/components/main-menu/main-menu.component.html [hunk 1]
```diff
@@ -81,6 +91,11 @@
 				routerLink="/learn">
 				{{resourceService?.frmelmnts?.tab?.courses}}
 			</a>
+			<a class="item" [class.active]="router.isActive('/search/Groups') || router.isActive('/groups')"
```
Comment (harishgilimi) [inline src/app/client/src/app/modules/core/components/main-menu/main-menu.component.html:94]: here also

File: src/app/client/src/app/modules/core/components/main-menu/main-menu.component.html [hunk 2]
```diff
@@ -10,6 +10,10 @@
 			appTelemetryInteract [telemetryInteractEdata]="learnMenuIntractEdata" routerLink="/learn">
 			{{resourceService?.frmelmnts?.tab?.courses}}
 		</a>
+		<a class="item py-8 pl-0 pr-24" [class.active]="router.isActive('/search/Groups') || router.isActive('/groups')"
```
Comment (harishgilimi) [inline src/app/client/src/app/modules/core/components/main-menu/main-menu.component.html:13]: what is this **/search/Groups** route ?

File: src/app/client/src/app/modules/core/components/main-menu/main-menu.component.html [hunk 3]
```diff
@@ -110,6 +125,12 @@
 			<a class="item" (click)="showSideBar()" *ngIf="router.isActive('', true)" routerLink="/explore">
 				{{resourceService.frmelmnts?.lbl?.explore}} {{resourceService.instance}}
 			</a>
+			<a class="item"
+				[class.active]="router.isActive('/search/groups') || router.isActive('/groups')"
```
Comment (harishgilimi) [inline src/app/client/src/app/modules/core/components/main-menu/main-menu.component.html:129]: what is this **/search/Groups** route ?

File: src/app/client/src/app/modules/core/components/main-menu/main-menu.component.html (hunk 2 above)
Comment (Nik720) [reply src/app/client/src/app/modules/core/components/main-menu/main-menu.component.html:13]: if groups can search then it should navigate to this route, I checked other menus and the same way I added.

File: src/app/client/src/app/modules/core/components/main-menu/main-menu.component.html (hunk 3 above)
Comment (harishgilimi) [reply src/app/client/src/app/modules/core/components/main-menu/main-menu.component.html:129]: This is not required now

**Pull request merged**